README.md

```diff
@@ -83,22 +157,26 @@ Testing Private Keys
 
 **Notice: Do not use these keys in production or put funds in them. They are for testing purposes only.**
 
+### How to Use
+
+1. Deploy contracts using the master account.
+2. Interact with contracts (e.g., mint NFTs, call functions) using one of the funded test accounts.
+3. Switch wallets in your web3 wallet (e.g., MetaMask) as needed.
+
+> **Tip:** Keep deployer and user accounts separate to better understand contract permissions and simulate real dApp usage.
+
 ### Contracts
-Before running contract commands, make sure you have a local Ethereum node running:
-- `git clone git clone https://github.com/OffchainLabs/nitro-devnode.git in apps` (TODO add a postinstall script to clone the repo)
-- `./apps/nitro-devnode/run-dev-node.sh` start a local Nitro node 
+Before you run contract commands, start a local Ethereum node:
+- Clone the devnode: `git clone https://github.com/OffchainLabs/nitro-devnode.git apps/nitro-devnode` (TODO: add a postinstall script to clone the repo)
```

Comment:
post install is already in place: https://github.com/ArbitrumFoundation/stylus-workshop-gol/blob/78527ae6c8e29346756f79721f10d2762270a0c0/package.json#L10